graylog2-web-interface/src/components/nodes/NodeMaintenanceDropdown.jsx

```diff
@@ -30,7 +31,7 @@ const NodeMaintenanceDropdown = React.createClass({
             </LinkContainer>
           </IfPermitted>
 
-          <MenuItem href={`${this.props.node.transport_address}api-browser`} target="_blank">
+          <MenuItem href={URLUtils.qualifyUrl('api-browser')} target="_blank">
```

Comment:
The link to the API browser is node-specific, as it is visually associated with a node element in the nodes list/part of the node details page of a specific node. The way it is changed here, makes it point to the API browser of the API endpoint node of the web interface, which might not be the same and is therefore counterintuitive. We should therefore disconnect the API browser link from the node element in the UI if it is not possible to reliably generate a link to the node's API browser.